v2/router.go

```diff
@@ -29,15 +29,15 @@ import (
 // TODO: Until golang/mock#4 is fixed, imports in the generated code have to
 // be fixed by hand. They use vendor/* import paths rather than direct.
 
-// Procedure specifies a single handler registered in the RouteTable.
-type Procedure struct {
+// TransportProcedure specifies a single transport-level handler registered in the RouteTable.
+type TransportProcedure struct {
```

Comment:
This will still require the changes to the plugins, right? i.e. the changes introduced in https://github.com/yarpc/yarpc-go/pull/1588

Reply:
Yup, just included those changes